packages/core/package.json

```diff
@@ -36,6 +36,7 @@
     "test:node": "mocha spec/*/*.spec.cjs && npm run test:node:esm",
     "test:node:esm": "mocha --experimental-json-modules spec/node/asciidoctor.spec.js",
     "test:browser": "node spec/browser/run.cjs",
+    "test:quickjs": "qjs --unhandled-rejection spec/quickjs/run.mjs",
```

Comment:
Could you please install/setup qjs on the CI and run `npm run test:quickjs`?

https://github.com/asciidoctor/asciidoctor.js/blob/d333b7bd74c4c435656f6bceb2e2f7782fbb259a/.github/workflows/build.yml#L24-L60

Reply:
I did it in my first draft, but the called npm target would fail since `qjs` would not be installed on the github runner. So I tried to run it on my own repo but [the CI](https://github.com/yne/asciidoctor.js/actions/runs/9531792112/job/26273129390) failed with
```
RuntimeError: To use Array#pack, you must first require 'corelib/array/pack'.
```
So I was stuck and decided to keep it out of CI for now

Reply:
> I did it in my first draft, but the called npm target would fail since qjs would not be installed on the github runner

I guess it's possible to install it on the GitHub runner?


> RuntimeError: To use Array#pack, you must first require 'corelib/array/pack'.

I will take a look next this week

Reply:
> I guess it's possible to install it on the GitHub runner?

Yes, it is now part of the Linux+MacOS runner

> I will take a look next this week

Nice, I started a CI while skipping the "corelib/array/pack" issue and the pipeline went fine
https://github.com/yne/asciidoctor.js/actions/runs/9734959295